feat: ROOT-208: More SDK endpoints coverage

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/18375232891)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/635
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18210200787

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18229287652

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-208'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18229287652

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18229560735

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18229838121

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18345141860)
> Successfully merged:  29 files changed, 1459 insertions(+), 221 deletions(-)

matt-bernstein committed:
Merge branch 'develop' into 'fb-ROOT-208'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18345141860

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18350320130

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-208'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18350320130

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18350608000

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18353989453

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18355236727

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-208'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18355236727

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18359155647

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-208'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18359155647

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18375163234

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-208'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18375163234

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18375232891

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/18377445319
